### PR TITLE
About that beer I owed ya. Adds blueshirt items to sectech and defines them better.

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: Adds blueshirt items to Sectech, and no longer do they have placeholder names and descriptions.

:cl: duckay
add: Added better names and descriptions for blueshirt
add: Added Blueshirt helmet and armor to the sectech for $600 each
/:cl:

[why]: Hotel has been mostly removed from the game, and so with it goes with its black mesa-esque security armor which I always liked, and the jumpsuit always bothered me as it always said it was a bland 'security jumpsuit' but instead it's unique. Snowflake time.

### CODE REVIEW

File: code/modules/vending/security.dm [hunk 1]
```diff
@@ -16,7 +16,10 @@
 	contraband = list(/obj/item/clothing/glasses/sunglasses = 2,
 		              /obj/item/storage/fancy/donut_box = 2)
 	premium = list(/obj/item/storage/belt/security/webbing = 5,
-					/obj/item/coin/antagtoken = 1)
+					/obj/item/coin/antagtoken = 1,
+					/obj/item/clothing/head/helmet/blueshirt = 1,
+					/obj/item/clothing/suit/armor/vest/blueshirt = 1)
+
```
Comment (subject217) [inline code/modules/vending/security.dm]: useless whitespace

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -84,8 +84,11 @@
 	tint = 2
 
 /obj/item/clothing/head/helmet/blueshirt
+	name = "blue helmet"
+	desc = "A reliable, blue tinted helmet, you can't but help to think of beer."
```
Comment (ATH1909) [inline code/modules/clothing/head/helmet.dm]: "can't but help"?

I feel like this could be worded better. Is it a reference to a specific quote?

Comment (duckay) [reply code/modules/clothing/head/helmet.dm]: I'll word it better anyway

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -84,8 +84,11 @@
 	tint = 2
 
 /obj/item/clothing/head/helmet/blueshirt
+	name = "blue helmet"
+	desc = "A reliable, blue tinted helmet. It reminds you of beer."
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
	desc = "A reliable, blue tinted helmet reminding you that you <i>still</i> owe that engineer a beer."
```

**Comment by duckay** [comment]
> Is 1.2K a fair price for this? I'd almost consider making it a set but I think the value should be mainly on the armor and less on the helmet.

It was made especially expensive because I originally thought people would be conflicted about putting items that usually belong in the armory into a vendor. 
I'll probably change it accordingly in a bit

**Pull request merged**